### PR TITLE
Updated dependencies of OSE Workshop 0.11.0

### PR DESCRIPTION
**Pull request by obivandamme**

### CODE REVIEW

**Comment by Olympic1** [comment]
I noticed that on GitHub v0.11.0 was already added back in october but not on KS, so the last downloadable version on CKAN is v0.10.2. Will you update KS in the future or do we need to switch to GH.

**Comment by obivandamme** [comment]
the version in october was a pre release. i removed that flag today. the
final version will be added to KS as soon as possible.
Am 17.11.2015 15:35 schrieb "Arne" <notifications@github.com>:

File: NetKAN/Workshop.netkan [hunk 1]
```diff
@@ -34,6 +34,17 @@
                     { "name": "KIS" }
                 ]
             }
+        },
+	{	
+            "version": ">=0.11.0",
+            "override": {
+                "depends": [
+                    { "name": "ModuleManager" },
+                    { "name": "CommunityResourcePack", "minVersion": "0.4.7" },
```
Comment (Olympic1) [inline NetKAN/Workshop.netkan]: `"min_version": "0.4.7.0"`

File: NetKAN/Workshop.netkan [hunk 2]
```diff
@@ -34,6 +34,17 @@
...
+		    { "name": "Firespitter Core" },
```
Comment (Olympic1) [inline NetKAN/Workshop.netkan]: The correct id is `FirespitterCore`, just remove the space

Comment (obivandamme) [inline NetKAN/Workshop.netkan]: oh, sorry that is a typo. is the syntax correct for the minversion of CRP
and to make sure every install of version 0.11 and up use the new
dependencies while older releases use the old dependencies?
Am 17.11.2015 16:15 schrieb "Arne" <notifications@github.com>:

Comment (Olympic1) [inline NetKAN/Workshop.netkan]: No, it needs to be "min_version": "0.4.7.0" for CRP

Comment (obivandamme) [inline NetKAN/Workshop.netkan]: ok i will fix that, too
Am 17.11.2015 16:34 schrieb "Arne" <notifications@github.com>:

**Comment by Olympic1** [comment]
Thanks, Merging

**Pull request merged**